test/conformance/parallel/ptp.go

```diff
@@ -163,7 +138,7 @@ var _ = Describe("[ptp-long-running]", func() {
 			for {
 				select {
 				case msg := <-messages:
-					if testParameters.SoakTestConfig.MasterOffsetConfig.FailFast {
+					if testParameters.SoakTestConfig.MasterOffsetConfig.TestSpec.FailureThreshold > 0 {
 						cancel()
 						Fail(msg)
 						break L1
```

Comment:
what is L1?

Reply:
add good label

Reply:
Sure, but this piece of code (offset implementation) would undergo a heavy change as we have a JIRA issue for its proper implementation.